docs/doxygen-user/tagging.dox

```diff
@@ -18,7 +18,13 @@ Once you have chosen to tag the file or the result, there are two more options:
 
 \image html tagging-2.PNG
 <br>
-You can create arbitrary tag names. Autopsy remembers your tag names from previous cases, so choose wisely. Choose a tag from the list you have created, or create a "New Tag".
+There are several default tag names:
+- Bookmark - Default tag for marking files of interest
+- CAT-1 through CAT-5 - For law enforcement use
+- Follow Up - Default tag for marking files to follow up on
+- Notable item - Default tag for indicating that an item should be marked as notable in the central repository
+
+You can also create arbitrary tag names. These tag names will be automatically saved for future use. Choose a tag from the list you have created, or create a "New Tag".
```

Comment:
"custom" probably is better than "arbitrary." The latter seems slightly negative.